ERCS/erc-0000.md

```diff
@@ -0,0 +1,176 @@
+---
+eip: 0000
+title: Crosschain Token Interface
+description: Minimal token interface for cross-chain transfers
+author: skeletor <skeletor@defi.sucks>, parti (@0xParticle), joxes <joxes@defi.sucks>, ng <ng@defi.sucks>, agus duha <agusduha@defi.sucks>, disco <0xdiscotech@defi.sucks>, gotzen <gotzen@defi.sucks>, 0age <0age@uniswap.org>, Mark Tyneway <mark@oplabs.co>, Zain Bacchus <zain@oplabs.co>, Matt Solomon <msolomon@oplabs.co>, Maurelian <maurelian@protonmail.ch>
+discussions-to: https://ethereum-magicians.org/t/erc-xxxx-crosschain-token-interface/21508
+status: Draft
+type: Standards Track
+category: ERC
```

Comment:
I wonder does this fall into the Interface category as per [EIP-1](https://eips.ethereum.org/EIPS/eip-1).

Reply:
I think this belongs to ERC, as it's an application-level standard. The interface category applies to language-level standards.

Reply:
Agreed 👍🏻